src/main/java/domain/state/State.java

```diff
@@ -0,0 +1,12 @@
+package domain.state;
+
+import domain.Pins;
+
+public interface State {
+
+    State bowl(Pins downPins);
+
+    Boolean isClosed();
```

Comment:
참조 자료형 대신 기본 자료형을 사용하면 어떨까요?

```suggestion
    boolean isClosed();
```

Reply:
적용